.env.example

```
@@ -297,4 +297,8 @@ CHROMADB_AUTH_TOKEN=            # Optional: Authentication token
 
 LOG_LEVEL=info                  # Logging level: debug, info, warn, error
 DEBUG_MEMORY=false              # Enable memory system debug logging
-DEBUG_STORAGE=false             # Enable storage adapter debug logging 
\ No newline at end of file
+DEBUG_STORAGE=false             # Enable storage adapter debug logging
+# Optional: export OpenInference AI traces to a configured Files SDK store
+AGENTPOND_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AGENTPOND_ENABLED key should go before the DEBUG_MEMORY key

(UnorderedKey)